components/list/README.md

```diff
@@ -347,6 +408,8 @@ The `d2l-list-item` provides the appropriate `listitem` semantics for children w
 | `drag-target-handle-only` | Boolean | Make the drag target the drag handle only. |
 | `drop-nested` | Boolean | Whether nested items can be dropped on this item |
 | `drop-text` | String | Text to drag and drop |
+| `expandable` | Boolean | Whether or not to show the expand/collapse toggle. |
+| `expanded` | Boolean | Default state for expand collapse toggle - if not set, collapsed will be the default state. Requires `expandable` to be set. |
```

Comment:
We're not entirely consistent with how we specify defaults in these tables, partly because we've changed over time, and because we've had a number of different authors, some not all from our team (like the list component). 

I think our current practice is to specify them, and to do it like this, however we would opt for being consistent in the current context (which implies the default rather than being explicit). Honestly, the entire list documentation needs an overhaul, so in this case I would just be consistent with what's there.

```markdown
| `expanded`, default: `false` | Boolean | Whether the item is expanded. Requires `expandable` to be set. |
```

I think it would be good to update the wording as I have above to be clear that this property controls the state, and it's not only for the default/initial state.

Reply:
To clarify, do we want to state `default: false` as you have in your example? It would not be consistent with current documentation so I assume we do not want to include that.

Reply:
Correct. I would exclude it for now given it would be inconsistent. Feel free to mention it in the description if you feel it's not implied. But I think we should update the description as mentioned.